### PR TITLE
Pip-info scripts added - Hazardouz

### PR DESCRIPTION
**Pull request by kevinjacb**

### CODE REVIEW

**Comment by Shivansh-007** [comment]
Lint before you push :D

File: hello_world/tabulate.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
```
Comment (Shivansh-007) [inline hello_world/tabulate.py:1]: This isn't required.

File: hello_world/tabulate.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+from typing import Any, List, Optional
+
+def make_table(raw_data: List[List[Any]],rows: List[List[Any]], labels: Optional[List[Any]] = None, centered: bool = False) -> str:
```
Comment (Shivansh-007) [inline hello_world/tabulate.py:4]: Remember next time, we are using python3.9 which includes `list`, `tuple` as a typehint. But no problem, I am fine with anything.

```suggestion
def make_table(raw_data: list[list[Any]],rows: list[list[Any]], labels: Optional[list[Any]] = None, centered: bool = False) -> str:
```

**Review by Shivansh-007 — CHANGES_REQUESTED** [review]

File: hello_world/pip_info.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from rich import print 
+import sys
+import os
+import tabulate
+
+# files = []
```
Comment (JoeNibe) [inline hello_world/pip_info.py:6]: remove comment

**Review by JoeNibe — CHANGES_REQUESTED** [review]
reviewed

File: hello_world/pip_info.py (hunk 3 above)
Comment (Shivansh-007) [reply hello_world/pip_info.py:6]: Any reason why? I don't see the need to do that.

File: hello_world/tabulate.py (hunk 2 above)
Comment (p0lygun) [reply hello_world/tabulate.py:4]: yh this should be changed

Comment (JoeNibe) [reply hello_world/tabulate.py:4]: well i think its not required but makes it easier as you dont have to import more stuff.

Comment (Shivansh-007) [reply hello_world/tabulate.py:4]: Yup, 

> But no problem, I am fine with anything.

File: hello_world/pip_info.py (hunk 3 above)
Comment (JoeNibe) [reply hello_world/pip_info.py:6]: just getting familiar with github. just wanted to check how suggestions work.

**Review by p0lygun — COMMENTED** [review]
just some linting issues that needs to be fixed

**Review by JoeNibe — APPROVED** [review]

**Review by Shivansh-007 — APPROVED** [review]

**Pull request merged**